com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/ShaderProgramBuilderTest.java

```diff
@@ -334,7 +334,7 @@ public void testShaderReflection() throws Exception {
 
             debugPrintShader("Reflection Test 2", shader);
 
-            assert(shader.getTexturesCount() == 9);
+            assertEquals(8, shader.getTexturesCount());
```

Comment:
`assert` function turned off by default in jvm. After replacing to `assertEquals` I found that the value is wrong here